Assets/SEE/UI/Window/TreeWindow/DesktopTreeWindow.cs

```diff
@@ -50,53 +56,116 @@ public partial class TreeWindow
         /// </summary>
         private static readonly GradientAlphaKey[] alphaKeys = { new(1, 0), new(1, 1) };
 
+        /// <summary>
+        /// The input field in which the user can enter a search term.
+        /// </summary>
+        private TMP_InputField SearchField;
+
+        /// <summary>
+        /// Orders the tree below the given <paramref name="orderBelow"/> node according to the graph hierarchy.
+        /// This needs to be called whenever the tree is expanded.
+        /// </summary>
+        /// <param name="orderBelow">The node below which the tree should be ordered.</param>
+        private void OrderTree(Node orderBelow)
+        {
+            Debug.Log("Ordering tree.");
```

Comment:
Debugging output could likely be removed, can it?

Reply:
Indeed, I forgot to remove it. Has now been fixed.